packages/design-system/src/components/Promo/Promo.js

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import { Button } from '../Button';
+
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'promo';
+
+export const Promo = props => {
+  const {
+    buttonText,
+    children,
+    className,
+    header,
+    image,
+    light,
+    linkText,
+    onButtonClick,
+    onClose,
+    onLinkClick,
+    size,
+    img,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}--light`]: light,
+      [`${baseClass}--${size}`]: size,
+      [`${baseClass}`]: true,
+    }),
+    className
+  );
+  const shouldRenderFooter = buttonText || linkText;
+
+  return (
+    <div className={mergedWrapperClassNames} {...restProps}>
+      <div className={styles[`${baseClass}__content`]}>
+        <img src={img} className={styles[`${baseClass}__content-img`]} />
```

Comment:
simplify those class names - if there is no other `...img` here you don't need to use whole "__content-img".
```suggestion
        <img src={img} className={styles[`${baseClass}__img`]} />
```

Reply:
✅